arc/src/arc_curie_pme.c

```diff
@@ -187,12 +190,13 @@ void curie_pme_end_restore_mode(void)
 }
 
 // mark -- getter and setters--
-PATTERN_MATCHING_DISTANCE_MODE curie_pme_get_distance_mode(void) // L1 or LSup
+PATTERN_MATCHING_DISTANCE_MODE curie_pme_get_distance_mode(void)  // L1 or LSup
 {
     return (GCR_DIST & reg_read16(GCR)) ? LSUP_DISTANCE : L1_DISTANCE;
 }
 
-void curie_pme_set_distance_mode(PATTERN_MATCHING_DISTANCE_MODE mode) // L1 or LSup
+void curie_pme_set_distance_mode(
+    PATTERN_MATCHING_DISTANCE_MODE mode)  // L1 or LSup
```

Comment:
OK now I see what the comment means I guess. I think it's dumb, just drop it. You look  up DISTANCE_MODE if you want to know what it means. Or add param documentation above. It's not clear enough this way.